README.md

```diff
@@ -60,26 +60,26 @@ trestle -h
 
 ## Supported OSCAL elements and extensions
 
-`trestle` implciltly supports all OSCAL schemas for use within the object model. As trestle a tentative roadmap includes
+`trestle` implcitly supports all OSCAL schemas for use within the object model. The development roadmap for `trestle` includes
```

Comment:
I guess there's still a type  in implicity.

Reply:
thanks - fixed